ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -4,16 +4,17 @@
 import unittest
 import os
 import sys
+import shutil
+import tempfile
 import logging
 from StringIO import StringIO as StringBuffer
 from mock import patch
-from ph5 import logger, ch as CH
 from ph5.utilities import segd2ph5, tabletokef
 from ph5.core import experiment, segdreader
-from ph5.core.tests.test_base import TempDirTestCase
+from ph5.core.tests.test_base import LogTestCase, import TempDirTestCase
```

Comment:
```
from ph5.core.tests.test_base import LogTestCase, TempDirTestCase
```

Reply:
I resolved this on the webpage. It was hard to check on the web. Then I pull to my computer to check. Check out the new commit